config/build/manager/manager.yaml

```diff
@@ -23,11 +23,9 @@ spec:
         control-plane: controller-manager
     spec:
       containers:
-      - command:
-        - /manager
-        args:
+      - args:
```

Comment:
Why did the args disappear in all 4 places?

Reply:
nvm, realized `ko` now rewrites the entrypoint for us

Reply:
ko sets the entry point for the container. Keeping the definition in the deployment creates a craslhoop because `/manager` isn't an executable in the container.